recommendations.md

```diff
@@ -13,9 +13,9 @@ access tokens (see (#insufficient_uri_validation)). It can also help to detect
 mix-up attacks (see (#mix_up)).
 
 Clients and AS MUST NOT expose URLs that forward the user's browser to
-arbitrary URIs obtained from a query parameter ("open redirector").
-Open redirectors can enable exfiltration of authorization codes and
-access tokens, see (#open_redirector_on_client).
+arbitrary URIs obtained from a query parameter ("open redirector") as
+described in (#open_redirection). Open redirectors can enable
```

Comment:
Isn't the internal link syntax curly quotes? `{{#open_redirection}}`

Reply:
Mmark wants parentheses: https://github.com/mmarkdown/mmark/blob/master/Syntax.md#cross-references